### PR TITLE
Add AnyLogic.app v7.1.2

### PR DESCRIPTION
**Pull request by denpatin**
Great tool! I'm suprised why it wasn't present as a cask!

AnyLogic is:

> - A multimethod simulation software
> - The only simulation tool that supports Discrete Event, Agent Based, and System Dynamics Simulation

There are several versions of this software but I have chosen the `Personal Learning Edition`, which is free for personal & self-educational use. If needed, I can also add `commercial` versions, but I am not sure how to name the casks for them.

### CODE REVIEW

File: Casks/anylogic.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  pkg 'Install AnyLogic.pkg'
+
+  caveats <<-EOS.undent
+    AnyLogic #{version.slice(/\w+/)} requires Java 8. You can install the latest Java using
+      brew cask install java
+  EOS
+
+  caveats <<-EOS.undent
+    For activation instructions, check up on
+      http://www.anylogic.com/upload/activation-guides/AnyLogic_#{version.slice(/\w+/)}_PLE_License.pdf
```
Comment (vitorgalvao) [inline Casks/anylogic.rb]: `#{version.slice(/\w+/)}` can be greatly simplified as `#{version.to_i}`.

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/phinze/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/phinze/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

Also, please unify the caveats.

**Comment by denpatin** [comment]
@vitorgalvao Thank you for help!

* Simplified the code you indicated
* Added the `uninstall` section
* Unified the caveats

And rebased the commit.

**Comment by vitorgalvao** [comment]
Looks good. Thank you.

**Pull request merged**